charts/celeborn/Chart.yaml

```diff
@@ -39,4 +39,4 @@ version: 0.1.0
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 1.16.0
+appVersion: 0.2.1
```

Comment:
appVersion should celeborn released version.

Reply:
Hi @zwangsheng , thanks for your PR! One of the users also encounters an issue after running helm upgrade, I wonder if you also encounter the same issue
![image](https://user-images.githubusercontent.com/948245/226841890-c9cfe0ff-79e8-4a11-8e9b-911844143462.png)
![image](https://user-images.githubusercontent.com/948245/226842476-cc2a8ce7-d036-4ee7-a33a-dc5cbb438eb4.png)

He added an annotation before running helm upgrade
![image](https://user-images.githubusercontent.com/948245/226842380-c13e214a-5c5c-44b2-b96e-f8e46eefd514.png)

Reply:
anyway, @RexXiong and @FMX is looking into it

Reply:
Can you add more info about celeborn master roll up started or failed.

Reply:
The issue I'm having is that the scrolling update with image changed is obviously failing to update

Reply:
I see, so these are different issues, @RexXiong will submit a PR later